### ISSUE TITLE
Hydro Thunder

### ISSUE THREAD

**Issue by Ted-bissada** [Rinnegatamante/flycast-vita-Compatibility#79]
hydro thunder v1.020 GDI

flycast version 0.1

i completed the first 3 races it seems to run well with no smc,
there are minor graphical glitches with the water near the rear of the boat disappearing 
but i think that may be a transparency sorting issue

menu fps is full 30 with graphical glitches on icons sometimes
in game it varies from 9 at worst to 18 at best depending on area, most frequent seems to be around 12-13fps

recommended ingame+
but with just slightly more performance would call it playable

**Comment by alan-morford** [comment]
> 
> 
> hydro thunder v1.020 GDI
> 
> flycast version 0.1
> 
> i completed the first 3 races it seems to run well with no smc,
> there are minor graphical glitches with the water near the rear of the boat disappearing
> but i think that may be a transparency sorting artifact
> 
> menu fps is full 30 with graphical glitches on icons sometimes
> in game full fps would be 30 it varies from 9 at worst to 18 at best depending on area,
> most frequent seems to be around 12-13fps
> 
> recommended ingame+
> but with just slightly more performance would call it playable

Can you explain "no smc"? I'm not getting playable results at all. Can you share the settings you have? Sound on? Graphical settings? I'd like to verify your test results.

**Comment by Ted-bissada** [comment]
smc is self modifying code checks aka just use the normal way to start instead of the other text when opening flycast and my results are like 1/2 speed so not really playable but almost

**Comment by alan-morford** [comment]
> 
> 
> smc is self modifying code checks aka just use the normal way to start instead of the other text when opening flycast and my results are like 1/2 speed so not really playable but almost

Oh I see it now. There's CPU modes. I'm slow. Can I assume there's a setting to change in Flycast to boot the game with SMC (Performance)? Or do I need to recompile it with that option enabled? I'm not a retroarch user if you can't tell.

**Comment by Ted-bissada** [comment]
when you open the flycast bubble there are three options just clicking start opens it in normal mode this is the fastest the other ones are only for if the game has self modifying code but they slow it down that clickable text off to the side is the other two options the one you want is just opening it normally with the square that says start
![image0](https://user-images.githubusercontent.com/20605185/104276251-850a9400-5472-11eb-8fb3-8d0c3a50bb46.jpg)

**Comment by alan-morford** [comment]
Do you happen to have your Vita overclocked? I'm playing the game in super slow motion. I know it's not ready for full play, just trying to duplicate your results. Thanks for the replies. Happy to keep testing other games for the cause. I'm excited to see how this emulator progresses.

**Comment by Ted-bissada** [comment]
yes to use this emulator setting the cpu at 500mhz is recommended, and this game is labeled ingame+ because it operates with minimal glitches but is considered too slow to play enjoyably maybe in a subsequent version this game will but fully playable this is version 0.1 afterall

**Comment by alan-morford** [comment]
> yes to use this emulator setting the cpu at 500mhz is recommended, and this game is labeled ingame+ because it operates with minimal glitches but is considered too slow to play enjoyably maybe in a subsequent version this game will but fully playable this is version 0.1 afterall

RIght I'm tracking that. Thanks again.
